### DISCUSSION TITLE
adding an automatic stop

### DISCUSSION THREAD

**Discussion by Jaydah1201346** [bonsai-rx/bonsai#1951] [General]
I am very new to bonsai and I was wondering if anyone knew how (if it is possible) to add an automatic stop my camera from recording after my delay or timer, I've tried doing a timer and delay but nothing works. I just need five minutes of recording and then automatically stops recording.

**Comment by bruno-f-cruz** [comment]
Hey and welcome!

In the future please try to follow the #864 guidelines and look for similar questions before opening a new one. I will leave you with a link to a previously answered similar question, and close this one. If you have any further issues, try to continue the discussion in one of the existing threads.

https://github.com/orgs/bonsai-rx/discussions/1921#discussioncomment-10063729

Thanks!
